### PR TITLE
#182 Add a link on the non-admin navbar (including on Member pages) for librarians to get to the admin pages if logged in.

### PR DESCRIPTION
**Pull request by kskilt**
If a Member is a librarian, display a link to the admin pages in the general navbar. Possibly to the left of the member name? E.g., "Go to Admin display," though there's probably a shorter way of saying it. Maybe "Admin" with an arrow or other graphic?

Possibly also do similar on the admin pages, linking to the Member pages? (E.g., "Go to Member display," or something shorter.)

Closes https://github.com/rubyforgood/circulate/issues/182

### CODE REVIEW

**Comment by kskilt** [comment]
Screen Shot 2020-09-25 at 2 37 36 PM

**Comment by kskilt** [comment]
Screen Shot 2020-09-25 at 2 38 17 PM

Screen Shot 2020-09-25 at 2 37 04 PM (2)

Screen Shot 2020-09-25 at 2 37 14 PM

Screen Shot 2020-09-25 at 2 37 24 PM

Screen Shot 2020-09-25 at 2 38 03 PM

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -51,8 +51,11 @@
             <ul class="nav">
               <li class="nav-item"><%= link_to "View Inventory", items_path %></li>
               <li class="nav-item"><%= link_to "Become a Member", signup_path %></li>
+              <% if current_user.admin?%>
```
Comment (gregblake) [inline app/views/layouts/application.html.erb:54]:
```suggestion
              <% if current_user.admin? %>
```

Can you also make a similar change on line 92, please?

**Review by gregblake — COMMENTED** [review]
These changes look good. Just a minor formatting request.

**Comment by kskilt** [comment]
Admin Dashboard mobile page has Members portal button
Screen Shot 2020-09-25 at 2 44 38 PM

**Review by gregblake — APPROVED** [review]

**Pull request merged**